.github/workflows/ci.yml

```diff
@@ -106,11 +106,21 @@ jobs:
       - name: Install dependencies
         run: make install-dev
 
-      - name: Install dependencies
-        run: ./scripts/deps --print-paths >> "$GITHUB_PATH"
+      - name: Build Docker image with test fixtures
+        run: docker build --build-arg TEST_BUILD=true -t launchpad-test .
 
-      - name: Coverage
-        run: make coverage
+      - name: Run coverage in Docker
+        run: |
+          docker run --rm \
+            -e LAUNCHPAD_ENV=development \
+            -e LAUNCHPAD_HOST=localhost \
+            -e LAUNCHPAD_PORT=2218 \
+            -e KAFKA_BOOTSTRAP_SERVERS="localhost:9092" \
+            -e KAFKA_GROUP_ID="launchpad-test" \
+            -e KAFKA_TOPICS="preprod-artifact-events" \
+            -v ${{ github.workspace }}:/workspace \
+            -w /workspace \
+            launchpad-test python -m pytest tests/unit/ tests/integration/ -v --cov --cov-branch --cov-report=xml --junitxml=junit.xml
```

Comment:
This happened to work before but now also needs to run in Docker.

Reply:
I made it work before on purpose, I think it would be nice if we don't have to run tests in Docker because I think that puts another step in the way of debugging test failures when they come up. However I guess there is a balance so if gets to the point we have to use docker I because we have to many weird tools thats reasonable but I think it should be a decision we take rather than the non-docker build just gradually eroding.